docs/KNOWNISSUES.md

```diff
@@ -2,138 +2,119 @@
 As a result, scripts that run on Windows may not run on non-Windows platforms because of the differences in the frameworks.
 For more information about .NET Core Framework, see <https://www.dotnetfoundation.org/netcore>
 
+With the advent of [.NET Standard 2.0](https://blogs.msdn.microsoft.com/dotnet/2016/09/26/introducing-net-standard/), .NET Core 2.0 will bring back many of the traditional types and methods present in the full .NET Framework.
+This means that PowerShell Core will be able to load many traditional Windows PowerShell modules without modification.
+You can follow our .NET Standard 2.0 related work [here](https://github.com/PowerShell/PowerShell/projects/4).
+
 ### Redirection Issues
 
 Input redirection is not supported in PowerShell on any platform.
 [Issue #1629](https://github.com/PowerShell/PowerShell/issues/1629)
 
-Use either `Get-Content` to write the contents of a file into the pipeline.
-
-PowerShell does not currently support "direct pipelining" external commands.
-Although the pipeline works properly for built-in PowerShell commands,
-with external (also called native) commands, each individual
-command in the pipeline is run to completion and then the aggregated
-data is passed to the next command.
-(This behavior is intended to be fixed in a later release.)
+Use `Get-Content` to write the contents of a file into the pipeline.
 
 Redirected output will contain the Unicode byte order mark (BOM) when the default UTF-8 encoding is used.
 The BOM will cause problems when working with utilities that do not expect it or when appending to a file.
-
-Use `-Encoding ascii` to write ASCII text (which, not being Unicode, will not have a BOM).
+Use `-Encoding Ascii` to write ASCII text (which, not being Unicode, will not have a BOM).
+(Note: see [RFC0020](https://github.com/PowerShell/PowerShell-RFC/issues/71) to give us feedback on improving the encoding experience for PowerShell Core across all platforms.
+We are working to support UTF-8 without a BOM and potentially changing the encoding defaults for various cmdlets across platforms.)
 
 ### Job Control
 
 There is no job-control support in PowerShell on Linux/macOS.
 The `fg` and `bg` commands are not available.
-`Ctrl-Z` sends the `powershell` process to the background.
+
+For the time being, you can use [PowerShell jobs](https://msdn.microsoft.com/en-us/powershell/reference/5.1/microsoft.powershell.core/about/about_jobs) which do work across all platforms.
 
 ### Remoting Support
 
-Client-side remoting from Linux/macOS is not supported with the initial package.
-The work is being done in the [psl-omi-provider](https://github.com/PowerShell/psl-omi-provider) repo.
+Currently, PowerShell Core supports PowerShell Remoting (PSRP) over WSMan with Basic authentication on macOS and Linux, and with NTLM-based authentication on Linux.
+(Kerberos-based authentication is not supported.)
```

Comment:
That doesn't mean it won't be planned to be supported. ;) 

In all fairness, the other stuff that does plan to be supported in this doc explicitly listed as such. I'd rather not rule something out entirely. If people want to tell us we need it (and they have, I still want to talk the psl-omi-provider folks), we should work it out.